.travis.yml

```diff
@@ -4,14 +4,9 @@ language: php
 
 matrix:
   include:
-    - php: 5.3
-      dist: precise
-    - php: 5.4
-    - php: 5.5
-    - php: 5.6
-    - php: 7.0
-    - php: 7.1
     - php: 7.2
+    - php: 7.3
+    - php: 7.4
```

Comment:
We migrated to GH Actions yesterday, check https://github.com/serbanghita/Mobile-Detect/blob/master/.github/workflows/devel.yml
Can you include PHP 8 as well in the matrix?